README.md

```diff
@@ -27,3 +27,7 @@ pip install -r requirements.txt
 cd src
 python2.7 -m main.py
 ```
+
+## How to use Engine Application
+1. For moving any vertex of Graph just "capture it" with left button mouse
+2. For connecting to Server open "Server Settings" and enter you credentials
```

Comment:
"2. For connecting a server go to "File" -> "Server settings" fill all necessary fields and press "Ok". Any server connection errors will be displayed in status bar at the top of the window."
Seems to be a little bit more informative.